junit-mock/src/lib.rs

```diff
@@ -69,15 +70,30 @@ pub struct Options {
     pub test_rerun: TestRerunOptions,
 }
 
-#[derive(Debug, Parser)]
+impl Default for Options {
+    fn default() -> Self {
+        Options::try_parse_from([""]).unwrap()
+    }
+}
+
+#[test]
+fn options_can_be_defaulted_without_panicing() {
+    Options::default();
+}
+
+#[derive(Debug, Parser, Clone)]
 #[group()]
 pub struct GlobalOptions {
     /// Seed for all generated data, defaults to randomly generated seed
     #[arg(long)]
     pub seed: Option<u64>,
+
+    /// Timestamp for all data to be based on, defaults to now
+    #[arg(long)]
+    pub timestamp: Option<DateTime<FixedOffset>>,
```

Comment:
This allows creating more deterministic mocks which is great for validation checking